traffic_control/models.py

```diff
@@ -595,3 +595,146 @@ class Meta:
 
     def __str__(self):
         return "%s %s %s" % (self.id, self.code, self.value)
+
+
+class Barrier(Enum):
+    BOOM = "BOOM"
+    FENCE = "FENCE"
+    FENCE_WITH_ARROW = "ARROWFENCE"
+    POST = "POST"
+    POST_LEFT_OF_LANE = "POST_LEFT"
+    POST_RIGHT_OF_LANE = "POST_RIGHT"
+    CONE = "CONE"
+
+    class Labels:
+        BOOM = _("boom")
```

Comment:
Minor thing, but let's make all labels start with uppercase for the sake of consistency.